### PR TITLE
PR: Change convention and refactor icons on the code analysis pane

### PR DESCRIPTION
**Pull request by juanis2112**
## Description of Changes

* [ ] Wrote at least one-line docstrings (for any new functions)
* [ ] Added unit test(s) covering the changes (if testable)

* [x] Included a screenshot or animation (if affecting the UI, see [Licecap](https://www.cockos.com/licecap/))

Change convention and refactor icons on the code analysis pane. 

Screen Shot 2020-04-23 at 4 08 45 AM

### Issue(s) Resolved

Fixes #

### Affirmation

By submitting this Pull Request or typing my (user)name below,
I affirm the [Developer Certificate of Origin](https://developercertificate.org)
with respect to all commits and content included in this PR,
and understand I am releasing the same under Spyder's MIT (Expat) license.

I certify the above statement is true and correct:

Juanis2112

### CODE REVIEW

File: spyder/utils/icon_manager.py [hunk 1]
```diff
@@ -279,8 +279,8 @@
     'copywop':                 [('fa.terminal',), {'color': MAIN_FG_COLOR}],
     'editpaste':               [('fa.paste',), {'color': MAIN_FG_COLOR}],
     'edit':                    [('fa.edit',), {'color': MAIN_FG_COLOR}],
-    'convention':              [('spyder.circle-letter-c',), {'color':'#3775a9'}],
-    'refactor':                [('spyder.circle-letter-r',), {'color':'#3775a9'}],
+    'convention':              [('mdi.alpha-c-box',), {'color':'#3775a9', 'scale_factor': BIG_ATTR_FACTOR}],
```
Comment (CAM-Gerlach) [inline spyder/utils/icon_manager.py:282]:
```suggestion
    'convention':              [('mdi.alpha-c-box',), {'color': '#3775a9', 'scale_factor': BIG_ATTR_FACTOR}],
```

File: spyder/utils/icon_manager.py [hunk 2]
```diff
@@ -279,8 +279,8 @@
...
+    'refactor':                [('mdi.alpha-r-box',), {'color':'#3775a9', 'scale_factor': BIG_ATTR_FACTOR}],
```
Comment (CAM-Gerlach) [inline spyder/utils/icon_manager.py:283]:
```suggestion
    'refactor':                [('mdi.alpha-r-box',), {'color': '#3775a9', 'scale_factor': BIG_ATTR_FACTOR}],
```

**Comment by ccordoba12** [comment]
@juanis2112, thanks a lot for your quick response about this!

@CAM-Gerlach, could you explain the reason behind your suggestions?

**Comment by CAM-Gerlach** [comment]
@ccordoba12 Just fixing the PEP8 issue with no space after the `:` that PEP8Speaks complained about. All the other lines had a space there so it made sense to fix it here since the line was being touched per the standard policy, whereas I didn't break the line length to fix the other PEP8 because that would be inconsistent with the rest of the code block since they were all one line and lots of others were >80 characters so it makes sense to just keep it.

**Pull request merged**